fw-child/resources/js/child-functions.js

```diff
@@ -691,13 +691,22 @@ const $ = jQuery;
     // Add 'scrolled' class on body after 50px scroll.
     //
 
-    $( window ).on( 'scroll', function() {
-      if ( $( this ).scrollTop() > 50 ) {
-        $( 'body' ).addClass( 'scrolled' );
+    function toggleBodyScrolledClass() {
+      if ($(window).scrollTop() > 50) {
+        $('body').addClass('scrolled');
       } else {
-        $( 'body' ).removeClass( 'scrolled' );
+        $('body').removeClass('scrolled');
       }
+    }
+
+    $( window ).on( 'scroll', function() {
+      toggleBodyScrolledClass();
     });
+
+    $( window ).on( 'load', function() {
+      toggleBodyScrolledClass();
+    });
```

Comment:
If the page and its files are already in cache, the "load" event may be fired before the browser executes this part of the script. In that case, since the "load" event was already fired, the function won't be executed.

I suggest to use the jQuery "ready" event which will fire once the DOM is loaded, and will fire even if the browser's native event `DOMContentLoaded` was already fired at the time of execution.

The recommended way of using this event is like so: `$(toggleBodyScrolledClass)`.